index.html

```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
     </div>
     
     <h3>Hello, yourName</h3>
+
+     <!--monthly income input  -->
+     <div class="goalsAndExpenses">
+        <label for="MonthlyIncome">Monthly Income</label>
+        <input class="form-control " name="income" id="monthlyIncome" value="" autofocus placeholder="Monthly Income">
+    </div>
+
+    <!--monthly savings goal input  -->
+    <div class="monthlySavings">
+        <label for="savingsGoal">Savings Goal</label>
+        <input class="form-control" type="savings" name="savingsGoal" id="savingsGoal" value="" placeholder="Monthly Savings Goal">
```

Comment:
please add 'required' attribute to input's opening tag

Reply:
Working on it now.